src/contrib/sort/sort.h

```diff
@@ -0,0 +1,33 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file Use external generic C library function
+ */
+
+#ifndef TVM_CONTRIB_SORT_SORT_H_
+#define TVM_CONTRIB_SORT_SORT_H_
+
+#include <tvm/runtime/registry.h>
+
+namespace tvm {
+namespace contrib {
+
+template<typename DType>
+struct SortElem {
+  DType value;
+  int32_t index;
+  static bool is_descend;
+
+  SortElem(DType v, int32_t i) {
```

Comment:
Define a comparator function instead to pass to the sort, instead of using this static bool